### PR TITLE
Fabo/2636 fix post withdraw behaviour

### LINKED ISSUE #2636: Total balance not updating automatically after withdrawing rewards
**Describe the bug**

Total balance in balance header not updating automatically after withdrawing rewards and tx being incorporated in a block

**To Reproduce**

Withdraw
watch header

**Expected behavior**

Header updating to reflect the new balance

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser 
 - Version 

**Additional context**

### PR DESCRIPTION
**Pull request by faboweb**
Closes #2636

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/vuex/modules/distribution.js [hunk 1]
```diff
@@ -87,10 +89,12 @@ export default ({ node }) => {
       }
     },
     async postMsgWithdrawDelegationReward({ dispatch }) {
-      await dispatch(`getTotalRewards`)
-      await dispatch(`getRewardsFromMyValidators`)
-      await dispatch(`queryWalletBalances`)
-      await dispatch(`getAllTxs`)
+      return Promise.all([
+        dispatch(`getTotalRewards`),
+        dispatch(`getRewardsFromMyValidators`, true),
```
Comment (jbibla) [inline src/vuex/modules/distribution.js]: lol what is `true`?

File: src/vuex/modules/distribution.js [hunk 2]
```diff
@@ -87,10 +89,12 @@ export default ({ node }) => {
       }
     },
     async postMsgWithdrawDelegationReward({ dispatch }) {
-      await dispatch(`getTotalRewards`)
-      await dispatch(`getRewardsFromMyValidators`)
-      await dispatch(`queryWalletBalances`)
-      await dispatch(`getAllTxs`)
+      return Promise.all([
```
Comment (jbibla) [inline src/vuex/modules/distribution.js]: why do promises work better than `async` `await`?

File: src/vuex/modules/distribution.js (hunk 1 above)
Comment (faboweb) [reply src/vuex/modules/distribution.js]: forced update, but I removed that.

File: src/vuex/modules/distribution.js (hunk 2 above)
Comment (faboweb) [reply src/vuex/modules/distribution.js]: `await`s will wait for each `dispatch` to have completed. `Promise.all` fires all `dispatch`es in parallel and waits for them all to complete.

**Pull request merged**